lib/node_modules/@stdlib/math/base/special/acoth/benchmark/benchmark.js

```diff
@@ -0,0 +1,79 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2018 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var bench = require( '@stdlib/bench' );
+var randu = require( '@stdlib/random/base/randu' );
+var isnan = require( '@stdlib/math/base/assert/is-nan' );
+var pkg = require( './../package.json' ).name;
+var acoth = require( './../lib' );
+
+
+// VARIABLES //
+
+var opts = {
+	'skip': ( typeof Math.acoth !== 'function' ) // eslint-disable-line stdlib/no-builtin-math
+};
+
+
+// MAIN //
+
+bench( pkg, function benchmark( b ) {
+	var x;
+	var y;
+	var i;
+
+	b.tic();
+	for ( i = 0; i < b.iterations; i++ ) {
+		x = ( randu()*100.0 ) + 1.0;
```

Comment:
```suggestion
		x = ( randu()*100.0 ) + 1.1;
```

`randu()` returns values on the interval `[0,1)`. In which case, need to add a value slightly greater than `1` in order to ensure that `y` is not `NaN`.